delta/html.py

```diff
@@ -258,11 +265,18 @@ def classes_check(op):
     return True
 
 
+def _open_new_tab(context: dict):
+    if 'behavior' in context:
+        return context['behavior'] == LinkBehavior.NEW_TAB
+    else:
+        return context.get('openNewTab')
```

Comment:
Could this work?
```
return context.get('behavior') == LinkBehavior.NEW_TAB or context.get('openNewTab')
```

Reply:
No. `behavior` can have other values. In that case, it will fall back to `openNewTab` key, which is not the expected behavior.